### ISSUE TITLE
Question on create the input for Random Forest Regression

### ISSUE THREAD

**Issue by ys31jp** [tensorflow/decision-forests#114]
I am just started to learn TensorFlow and Keras, and these might be simple questions, but I am not able to get the ideas.

I have trained a Random Forest Regression model.

As previously, the input is reading from pandas csv and convert to the input throught this: tf_dataset_train = tfdf.keras.pd_dataframe_to_tf_dataset(train_df, label="y", task=tfdf.keras.Task.REGRESSION,)

And the input is: <PrefetchDataset element_spec=({'x1': TensorSpec(shape=(None,), dtype=tf.int64, name=None), 'x2': TensorSpec(shape=(None,), dtype=tf.int64, name=None), 'x3': TensorSpec(shape=(None,), dtype=tf.int64, name=None), 'x4': TensorSpec(shape=(None,), dtype=tf.float64, name=None), 'x5': TensorSpec(shape=(None,), dtype=tf.float64, name=None)}, TensorSpec(shape=(None,), dtype=tf.int64, name=None))>

![image](https://user-images.githubusercontent.com/83806490/172377920-ea35b268-2541-4a60-ae57-d5389dd5baee.png)

I don't know how to create this type of input again. For example, I have this and want to be the input of the model: x1=24, x2=99636, x3=45123, x4=10.81, x5=45.29

Because I will have this kind of error: 

![image](https://user-images.githubusercontent.com/83806490/172377980-97ad86b2-bcf0-45e0-81f2-87992b2d15f5.png)
